Superbuild/External_GLEW.cmake

```diff
@@ -1,7 +1,13 @@
 
 set(proj GLEW)
 
-set(${proj}_DEPENDENCIES "")
+# Set dependency list
+set(${proj}_DEPENDS
+  ""
+  )
```

Comment:
```suggestion
```

Since the parameter `DEPENDS_VAR` is specified, there is no need to set `${proj}_DEPENDS`, the current setting of `${proj}_DEPENDENCIES` is enough